### DISCUSSION TITLE
Window.phantom undefined when using in Nextjs

### DISCUSSION THREAD

**Discussion by ntchungse** [phantom/docs#238] [Help & Support]
### Summary

I am developing a dapp that using phantom to connect. Following docs, I use window.phantom but It's undefined. I also try wallet-adapter library but error WalletNotReady. I already installed phantom wallet and try run another projects Reactjs and it return window.phantom value. Is this a problem with phantom and Nextjs?

Update:
It seems like phantom not working in localhost.

### Steps to Reproduce

<img width="699" alt="Screenshot 2024-05-28 at 09 07 51" src="https://github.com/phantom/docs/assets/46298605/39e74d44-2b29-4efd-8578-c52f2570e743">

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @ntchungse,

Would you mind sharing your code for detecting the provider and establishing a connection?

**Reply by ntchungse** [reply]
I am just following the docs https://docs.phantom.app/solana/integrating-phantom

**Reply by adamdelphantom** [reply]
@ntchungse We have a number of dapps using nextjs. And I haven't had any other reports. If you'd like to share your nextjs specific code I can try to repro.
